### DISCUSSION TITLE
Filters doesn't work when localization and auto option is ON

### DISCUSSION THREAD

**Discussion by dhruv-tank** [microsoft-search/pnp-modern-search#1752] [Q&A]
Hi,

I have enabled localization and added the Auto option before the taxonomy field and I am able to display proper readable tags in the filter and list view. But when I filter using those tags the results is zero, even though there are items tagged and are visible in the list view.

I am able to filter using tags when localization is OFF.

Thanks,
Dhruv

**Comment by FranckyC** [comment]
Hi @dhruv-tank, first of all, please use the provided issue template...We don't even know what version you are talking about neither the managed property configuration you are using.

By any chance, do you use the `owstaxidmetadataalltagsinfo` property? If it is the case, this is a known bug that should be fixed in the `develop` branch. As a workaround, use a `RefinableStringXXX` property instead.

**Comment by dhruv-tank** [comment]
PnP Modern Search Web Parts v4.5.3.0

I tried creating a refinable string and mapped it with the owstaxidmetadataalltagsinfo property, still the filter doesn't work and shows 0 results.

If possible can you please provide steps or documentation or any existing post where it shows it is working?
